### DISCUSSION TITLE
Import CSV not showing push messages or downloading failures

### DISCUSSION THREAD

**Discussion by wpas81** [filamentphp/filament#12402] [Help]
### Package

Other

### Package Version

latest

### How can we help you?

Hi

I've created an Importer. This works fine: it uploads the file, show the "we're importing x lines in the background" messages and processes it. However it does not show a new popup when it's finished or triggers an download with the errors.

I have no experience with jobs & queues in Laravel so I guess there's something wrong (I did start the queue:work process before importing the csv)

thanks for your advice

**Comment by micraux** [comment]
Hi,

Did you add `->databaseNotifications()` in you PanelProvider?

**Comment by wpas81** [comment]
I guess I was tired and missed the notification-icon on the top.... 🤦

**Comment by wpas81** [comment]
Ok this is weird. It was all working just fine, made multiple importers all functional. 
So my Filament backend is ready and I'm ready to move to the frontend for which I'll use Jetstream... So I create a new project, add jetstream, run my migrations & add Filament again + copy my Filament files.. and now my imports won't work anymore... just as stated in my  post above: I can upload a file, match the columns but  the job won't execute?! Why?!
